pkg/apis/clusterregistry/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -86,72 +86,68 @@ var (
 				OpenAPIV3Schema: &v1beta1.JSONSchemaProps{
 					Type: "object",
 					Properties: map[string]v1beta1.JSONSchemaProps{
-						"apiVersion": {
+						"apiVersion": v1beta1.JSONSchemaProps{
```

Comment:
I think you need to run `gofmt -s` against this file.

Reply:
sure, will fix this.

Reply:
@perotinus this file is generated automatically, I think we do not need to update this?

Reply:
Done